tornado_opentracing/__init__.py

```diff
@@ -15,3 +15,5 @@
 from .initialization import init_tracing  # noqa
 from .initialization import init_client_tracing  # noqa
 from .tracing import TornadoTracing  # noqa
+from .scope_manager import ScopeManager  # noqa
```

Comment:
Is this used by anything other than tests?  Presumably tracers would be initialized before configuring this instrumentation so I'm not sure why it's necessary considering it doesn't implement or alter a scope manager itself.  Maybe just provide as a test helper?

Reply:
It's made available to end users when constructing tracers manually but no reason to make import available from root package. It can be imported from tornado_opentracing.scope_manager instead. Removed.